src/types/index.ts

```diff
@@ -13,6 +13,7 @@ type GroundPlace = StopGroup | StopCluster;
 type GroundPlaceActionHistory = Record<Gpuid, GroundPlaceActionOptions>;
 
 interface GroundPlaceBase {
+  [key: string]: unknown;
```

Comment:
what is the use of this ?

Reply:
Hmm, that was because the `removeUndefinedValues` method use with an object structured as a `GroundPlaceBase` have failed the TypeScript check.

But I have update it now, with a type generic for the `removeUndefinedValues` method 👍 
```ts
const removeUndefinedValues = <T>(object: T): void => { ... }
```